### PR TITLE
default to new visualizer URL in @xstate/inspect

### PR DESCRIPTION
**Pull request by jebeck**
As requested in #2542, here's a small change to update the URL for the inspector to the new visualizer. After working with the new viz further and seeing some of its issues, like [this one](https://github.com/statelyai/xstate-viz/issues/227), but also some general fragility*, I think my suggestion would be that you publish this change to @xstate/inspect under a @beta or @next tag so that the beta status of the new viz is clear to users? I have suggested this as well in the docs change in #2579.

*Fragility = times when the visualization stopped showing up, just a blank page. Unfortunately I didn't have time to stop and capture the instances when this happened by extracting the machines into sandboxes... but I'll try to find the time in the future.

### CODE REVIEW

File: packages/xstate-inspect/src/browser.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export function createDevTools(): XStateDevInterface {
 }
 
 const defaultInspectorOptions: InspectorOptions = {
-  url: 'https://statecharts.io/inspect',
+  url: 'https://stately.ai/viz',
```
Comment (Andarist) [inline packages/xstate-inspect/src/browser.ts:64]: We'll be adjusting the public URL this week - probably to something like https://stately.ai/inspect . So until then, I would hold off merging this PR

**Comment by Andarist** [comment]
> I think my suggestion would be that you publish this change to @xstate/inspect under a @beta or @next tag so that the beta status of the new viz is clear to users

I'm afraid that this is not easy to do with our current releasing pipeline. From what I understand the #2542 got fixed and is no longer an issue. We are not aware of any substantial problems with the new inspector so I think it's rather OK to just publish a new version of the `@xstate/inspect` as the latest version. If issues arise when people start using it more we'll just have to deal with them and fix those issues promptly

> *Fragility = times when the visualization stopped showing up, just a blank page. Unfortunately I didn't have time to stop and capture the instances when this happened by extracting the machines into sandboxes... but I'll try to find the time in the future.

It would be highly appreciated if you could try to get exact repro steps for this.

**Comment by jebeck** [comment]
> > I think my suggestion would be that you publish this change to @xstate/inspect under a @beta or @next tag so that the beta status of the new viz is clear to users
> 
> I'm afraid that this is not easy to do with our current releasing pipeline. From what I understand the #2542 got fixed and is no longer an issue. We are not aware of any substantial problems with the new inspector so I think it's rather OK to just publish a new version of the `@xstate/inspect` as the latest version. If issues arise when people start using it more we'll just have to deal with them and fix those issues promptly

Sure, it was just a suggestion. And partly it is also feedback: I don't find the new visualizer adequate to my needs, and I'm still heavily using the old one. The layout is still much more readable in the old visualizer (related to [xstate-viz #227](https://github.com/statelyai/xstate-viz/issues/227) where there are lingering weird edges after the fix...I know further improvements are planned!), and some other things are better, like I'm much more likely to see the initial cascade of events that fire when my app loads in the old visualizer, and that's one of my most frequent sites of debugging. (If I understand correctly, @davidkpiano confirmed that events that don't make it to the visualizer on load are due to time to create connection to the inspector, and I guess the difference between old and new is that new takes longer to make that connection and misses more events, at least in my recent experience.)

Overall, my biggest suggestion is that stately provide clearer documentation about what the statuses of the old vs. new visualizers are. Just a few lines in the @xstate/inspect README with the URLs for both and some verbiage to the effect of "old viz not being developed further, existing bugs won't be fixed; new viz is new and has some bugs, please file as you encounter" (ideally w/link to a template code sandbox for the new inspector) would be super helpful for setting expectations and letting us users help ourselves. (I don't love having to go to a Discord to get questions of this sort answered, and I think self-serve documentation would make it easier for you all as well.)

> 
> > *Fragility = times when the visualization stopped showing up, just a blank page. Unfortunately I didn't have time to stop and capture the instances when this happened by extracting the machines into sandboxes... but I'll try to find the time in the future.
> 
> It would be highly appreciated if you could try to get exact repro steps for this.

I understand that exact repro is desired, and I will try to make adjustments in my own codebases/workflow to make it easier for me to extract them in the future. At present it requires a lot of replacing actions and such with stubs and careful reading to make sure I don't leak any company stuff.

And let me be clearer about _why_ I report things without a repro sometimes, in case this is a source of frustration for y'all. It's *not* because I expect a magic fix! I understand it's very difficult to fix something without exact steps to reproduce, and I don't expect a vague bug report to even result in an _effort_ to try to fix something, really. Rather, what I hope is that my report of frequent problems with the new tool is useful feedback in a general sense—i.e., that some feedback is better than no feedback.

**Comment by davidkpiano** [comment]
This is now outdated as https://github.com/statelyai/inspect is now recommended for XState v5+.

**Pull request closed without merging**